### PR TITLE
bpo-33726: Add short descriptions to PEP references in seealso

### PR DESCRIPTION
**Pull request by andresdelfino**
https://bugs.python.org/issue33726

### CODE REVIEW

File: Doc/reference/compound_stmts.rst
Comment (willingc) [inline Doc/reference/compound_stmts.rst]: Perhaps:

The proposal that changed the declaration of metaclasses to the current syntax, and the semantics...

File: Doc/reference/compound_stmts.rst
Comment (willingc) [inline Doc/reference/compound_stmts.rst]: Perhaps:

The proposal that added class decorators. Function and method decorators were introduced earlier in :pep:`318`.

File: Doc/reference/compound_stmts.rst
Comment (willingc) [inline Doc/reference/compound_stmts.rst:816]: ...Python, and added supporting

File: Doc/reference/simple_stmts.rst
Comment (willingc) [inline Doc/reference/simple_stmts.rst]: The proposal that added syntax for annotating the types of variables (including class variables and instance variables), instead of expressing them through comments.

File: Doc/reference/simple_stmts.rst
Comment (willingc) [inline Doc/reference/simple_stmts.rst]: ...module to provide a standard syntax for type annotations that can be used in static analysis tools and IDEs.

**Review by willingc — COMMENTED** [review]
Thanks @andresdelfino. I've made a few suggestions that I believe give the reader more clarity on some of the PEPs. Please take a look and edit as you see best.

**Comment by andresdelfino** [comment]
@willingc all your suggestions seemed right to me, and I have included them verbatim.

**Review by willingc — APPROVED** [review]
Thanks @andresdelfino. This looks good to me.

@serhiy-storchaka or @Mariatta would either of you be able to take a look too. I want to make sure before I merge that I'm doing the backport correctly (I'm assuming the bot will do it, but want to double check). Thanks.

**Review by Mariatta — APPROVED** [review]

**Comment by Mariatta** [comment]
Yep, everything looks good. Ready to merge, @willingc. Miss-islington will backport :)

**Comment by serhiy-storchaka** [comment]
LGTM if it LGT you.

Just don't forget to edit the commit message after pressing the "**Squash and merge**" button.

**Pull request merged**
